source/edge-server/configure.txt

```diff
@@ -496,4 +421,143 @@ Upgrade Edge Server Procedure
 
       .. code-block::
 
-         make clean up
+         edgectl clean
+
+      Then, run the Edge Server again.
+
+      .. code-block::
+
+         edgectl start
+
+   .. step:: Remove Unneeded Files
+
+      After upgrading, you can remove any files that are no longer needed.
+
+      .. code-block::
+
+         edgectl prune
+
+.. _edge-server-cycle-api-key:
+
+Cycle the Edge Server API Key
+-----------------------------
+
+To cycle the Edge Server API key:
+
+.. procedure::
+
+   .. step:: Generate a New Token in Atlas
+
+      You can generate a new Edge Server token from the 
+      :ref:`edge-server-details`. Use the :guilabel:`Regenerate Token`
+      button to get a new token, and copy it. You won't be able to see this
+      token again, and must generate a new token if you do not copy it.
+
+   .. step:: Supply the New Edge Server Token
+
+      Use ``edgectl`` to be prompted for a new token from the Edge Server 
+      instance:
+
+      .. code-block:: shell
+
+         edgectl --update-token=true
+
+      This triggers ``edgectl`` to prompt you for the token. When you provide 
+      it, the Edge Server instance exchanges this token for a secret. It uses 
+      the secret to authenticate with Atlas when it syncs data.
+
+      .. important:: Required network access
+
+         To exchange the token for an API key, ``edgectl`` must have network
+         access. You cannot cycle the API key if ``edgectl`` is not online.
+
+.. _specify-edge-server-query:
+
+Specify the Edge Server's Query
+-------------------------------
+
+You can use the ``--query`` argument with the ``edgectl configure`` command
+to specify the Edge Server instance Sync query.
+
+The ``--query`` for an Edge Server instance defaults to ``"*"``, which 
+automatically syncs all data in all of the collections in your 
+:ref:`schema <schemas>`.
+
+If you would like to filter the data that syncs to your Edge Server, you have
+two options:
+
+- Pass a simple string query. For example, ``--query="truepredicate"``.
```

Comment:
I don't think this would be valid since there isn't a table name specified for the query. But @sudssm is working on this and can probably provide more details later.